script/build-fuzzers

```diff
@@ -39,34 +39,39 @@ for lang in ${languages[@]}; do
 
   lang_scanner="${lang_dir}/src/scanner"
   if [ -e "${lang_scanner}.cc" ]; then
-    $CXX $CXXFLAGS -g -O1 "-I${lang_dir}/src" -c "${lang_scanner}.cc" -o "${lang_scanner}.o"
+    $CXX "$CXXFLAGS" -g -O1 "-I${lang_dir}/src" -c "${lang_scanner}.cc" -o "${lang_scanner}.o"
     objects+=("${lang_scanner}.o")
   elif [ -e "${lang_scanner}.c" ]; then
-    $CC $CFLAGS -std=c99 -g -O1 "-I${lang_dir}/src" -c "${lang_scanner}.c" -o "${lang_scanner}.o"
+    $CC "$CFLAGS" -std=c99 -g -O1 "-I${lang_dir}/src" -c "${lang_scanner}.c" -o "${lang_scanner}.o"
     objects+=("${lang_scanner}.o")
   fi
 
 
   # Compiling with -O0 speeds up the build dramatically
-  $CC $CFLAGS -g -O0 "-I${lang_dir}/src" "${lang_dir}/src/parser.c" -c -o "${lang_dir}/src/parser.o"
+  $CC "$CFLAGS" -g -O0 "-I${lang_dir}/src" "${lang_dir}/src/parser.c" -c -o "${lang_dir}/src/parser.o"
   objects+=("${lang_dir}/src/parser.o")
 
   highlights_filename="${lang_dir}/queries/highlights.scm"
   if [ -e "${highlights_filename}" ]; then
     ts_lang_query_filename="${lang}.scm"
-    cp "${highlights_filename}" "out/${ts_lang_query_filename}"
+    cp "${highlights_filename}" "test/fuzz/out/${ts_lang_query_filename}"
   else
     ts_lang_query_filename=""
   fi
 
   # FIXME: We should extract the grammar name from grammar.js. Use the name of
   # the directory instead. Also, the grammar name needs to be a valid C
   # identifier so replace any '-' characters
-  ts_lang="tree_sitter_$(echo $lang | tr -- - _)"
-  $CXX $CXXFLAGS -std=c++11 -I lib/include -D TS_LANG="$ts_lang" -D TS_LANG_QUERY_FILENAME="\"${ts_lang_query_filename}\"" \
+  ts_lang="tree_sitter_$(echo "$lang" | tr -- - _)"
+  $CXX "$CXXFLAGS" -std=c++11 -I lib/include -D TS_LANG="$ts_lang" -D TS_LANG_QUERY_FILENAME="\"${ts_lang_query_filename}\"" \
     "test/fuzz/fuzzer.cc" "${objects[@]}" \
-    libtree-sitter.a "$LIB_FUZZER_PATH" \
-    -o "out/${lang}_fuzzer"
+    libtree-sitter.a \
+    -o "test/fuzz/out/${lang}_fuzzer"
 
-  python test/fuzz/gen-dict.py "${lang_dir}/src/grammar.json" > "out/$lang.dict"
+  # python test/fuzz/gen-dict.py "${lang_dir}/src/grammar.json" > "out/$lang.dict"
+  JQ_FILTER='.. | if .type? == "STRING" or (.type? == "ALIAS" and .named? == false) then .value else null end'
+  jq "$JQ_FILTER" < "${lang_dir}/src/grammar.json" |
+		grep -v "\\\\" | grep -v null |
+		iconv -c -f UTF-8 -t ASCII//TRANSLIT |
+		awk '!/^""$/' >"test/fuzz/out/$lang.dict"
```

Comment:
Are you sure that this shell pipeline makes the same as the deleted script?

Reply:
The python script made output sorted and reencoded strings to escaped byte strings like:
```
"\x7b"
"\x7c"
"\x7c\x3d"
"\x7c\x7c"
```

Reply:
Yeah it's used here https://github.com/vigoux/tree-sitter-fuzz-action/blob/main/entrypoint.sh

The jq filter here is better as it removes empty strings and libfuzzer doesn't allow utf8 input, just ascii, for core languages it's fine but for grammars that use funky symbols (julia, dhall), it won't work

Reply:
I don't know what's wrong with two backslash tokens, so why it's needed to drop them `grep -v "\\\\"` and what didn't   do with the python script.

But all above shell pipeline can be replaces with a single `jq` call:
```
< ../tree-sitter-c/src/grammar.json jq '
  [ ..
    | if .type? == "STRING" or (.type? == "ALIAS" and .named? == false) then .value else null end
    | select(. != null and test("\\S"))
  ] | unique | .[]
'
```

Reply:
To be honest I can't remember

But does your version strip special utf8 chars?

Reply:
The bellow would filter all non ascii:
```
< ../tree-sitter-dhall/src/grammar.json jq '
  [ ..
    | if .type? == "STRING" or (.type? == "ALIAS" and .named? == false) then .value else empty end
    | select(test("\\S") and length == utf8bytelength)
  ] | unique | .[]
'
```

Reply:
The original pipelined version doesn't remove duplicates and don't sort an input.

Reply:
that's better, thanks!